composer.json

```diff
@@ -39,10 +38,8 @@
         }
     },
     "scripts": {
-        "unit-tests": "vendor/bin/codecept run unit --coverage",
-        "bc-check": "vendor/bin/roave-backward-compatibility-check",
```

Comment:
I understand this library prevents further upgrade, I am ok to remove it temporarily.